README.md

```diff
@@ -70,129 +77,170 @@ for(let x = 0; x < 100; x++) {
 ```
 
 ## In-depth list of havoc to be caused
+
 ### JSON
+
 - JSON.parse(): modifies values depending on their type
-    - if value is a string, replace all normal spaces with thin spaces
-    - if value is a number, increment or decrement the value
-    - if value is a boolean, negate it
-    - if value is not supported, it stays as-is
-    ```
-    const stringifiedJson = JSON.stringify({
-        stringSample: "Enflaked Sample",
-        numberSample: 100,
-        booleanSample: true
-    })
-    console.log(JSON.parse(stringifiedJson))
-    // expectation: {
-    //  stringSample: "Enflaked Sample",
-    //  numberSample: 100,
-    //  booleanSample: true
-    //}
-
-    // enflaked result: {
-        // this looks like the same string but the space is not really a space but just one that looks like a space.
-    //      stringSample: 'Enflaked Sample',
-    //      numberSample: 99,
-    //      booleanSample: false
-    //  }
-    ```
+  - if value is a string, replace all normal spaces with thin spaces
+  - if value is a number, increment or decrement the value
+  - if value is a boolean, negate it
+  - if value is not supported, it stays as-is
+
+```js
```

Comment:
Let's not try to touch indentations for README- I think your default formatter is doing this